app/models/piece.rb

```diff
@@ -11,6 +11,29 @@ class Piece < ActiveRecord::Base
 
   # Instance Methods
 
+  def move_to(end_coord)
+
+    #Check if move is vaild for Piece
+    return false unless self.valid_move?(end_coord)
+
+    #Capture logic
+    game.uncaptured_pieces.each do |piece|
+      if end_coord == piece.coordinate
+        if self.color != piece.color
+          piece.destroy
```

Comment:
Ok I guess even more forward-thinking than that would be (a) Many chess apps have the ability to undo your move within a certain time frame and (b) many chess apps have a "move history" feature that would allow you to see old moves and captures. This might be hard to implement down the road if we destroy the piece from the db. My vote would be just to set the piece to captured and don't display it on the board. It's not like we're going to run out of room in the db from 40 pieces lol